packages/broker/src/broker.ts

```diff
@@ -48,9 +48,8 @@ export const createBroker = async (configWithoutDefaults: Config): Promise<Broke
             const mnemonic = generateMnemonicFromAddress(toEthereumAddress(brokerAddress))
 
             logger.info(`Welcome to the Streamr Network. Your node's generated name is ${mnemonic}.`)
-            logger.info(`View your node in the Network Explorer: https://streamr.network/network-explorer/nodes/${encodeURIComponent(nodeId)}`)
             logger.info(`Network node ${nodeId} running`)
-            logger.info(`Ethereum address ${brokerAddress}`)
+            logger.info(`Node address ${brokerAddress}`)
```

Comment:
We used to have an emphasis on the wallet (Polygon) address being the identity of the node. This is no longer the case, since `nodeId` is no longer associated with the wallet address but is its own thing entirely. Thus I think we should add the word `wallet` here to make that distinction clear, dunno do what do you think?
```suggestion
            logger.info(`Node wallet address ${brokerAddress}`)
```

Reply:
It's all a bit subjective but I'll add my perspective:
In the Docs/Hub we've been using `Wallet` in the context of the Owner wallet, where the account has moreso of a value holding agenda. The Node address has moreso of a signing agenda, so I think `address` or `account` is more appropriate. Since the terminology we use in our Docs onboarding and the Hub is `Node address` I think we should adopt it here too.

Reply:
Makes sense